<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_sponso</class>
 <widget class="QDialog" name="Dialog_sponso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_Fermer">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>650</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 15px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(189, 96, 96, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_BGSponsoring">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/Image/Images/BGSponsoring.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="gB_infopersonnel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>611</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgba(199, 153, 207, 200);
background: rgba(199, 153, 207, 100); 
border-radius: 10px;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEdit_Description">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 15px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 15px;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 15px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 15px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 15px;
</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_NumTel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 15px;
</string>
    </property>
    <property name="text">
     <string>Num√©ro tel :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_NumTel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 15px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 15px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Description">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 15px;
</string>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MP4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat SemiBold</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 15px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(207, 188, 153, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
    </property>
    <property name="text">
     <string>MP4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_VideoMP4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox_ID">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>120</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    selection-background-color: #111;
    selection-color: yellow;
    color: white;
    background-color: rgba(134, 47, 147, 50);
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
    padding: 1px 0px 1px 20px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_Supprimer">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>230</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 15px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_Modifier">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>190</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 15px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_Ajouter">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 15px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLabel" name="labe_TitlePerso">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>LEMON MILK</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(134, 47, 147,50);
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;</string>
   </property>
   <property name="text">
    <string>GESTION DES SPONSORS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tab_Spornors">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>761</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(199, 153, 207, 100);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_TirerParNom">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>610</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Trier par num tel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_TirerParID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Trier par ID</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_Sponso">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>300</y>
     <width>341</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(199, 153, 207, 100);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SupprimerS">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>610</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Info">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>80</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(134, 47, 147, 25);
	border: 2px solid rgba(134, 47, 147, 100);
		border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_BGSponsoring</zorder>
  <zorder>pushButton_Fermer</zorder>
  <zorder>gB_infopersonnel</zorder>
  <zorder>comboBox_ID</zorder>
  <zorder>pB_Supprimer</zorder>
  <zorder>pB_Modifier</zorder>
  <zorder>pB_Ajouter</zorder>
  <zorder>labe_TitlePerso</zorder>
  <zorder>tab_Spornors</zorder>
  <zorder>pB_TirerParNom</zorder>
  <zorder>pB_TirerParID</zorder>
  <zorder>textBrowser_Sponso</zorder>
  <zorder>pushButton_SupprimerS</zorder>
  <zorder>label_Info</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
